code/physics/physics.h

```diff
@@ -32,7 +32,7 @@
 #define PF_GLIDING				(1 << 14)
 #define PF_FORCE_GLIDE			(1 << 15)
 #define PF_NEWTONIAN_DAMP		(1 << 16)	// SUSHI: Whether or not to use newtonian dampening
-#define PF_NO_DAMP				(1 << 17)	// Goober5000 - don't damp velocity changes in physics; used for instantaneous acceleration
```

Comment:
Per above comment, the `PF_NO_DAMP` flag ought to be usable for more than just maneuvers, just like the `PF_NEWTONIAN_DAMP` flag is.

Reply:
I would prefer a separate flag for any new functionality like this. Otherwise, `player-not-use-ai` would be wiping an unrelated flag. It is only because it is currently only used by maneuvers that it can be safely reset by `player-not-use-ai`.

Reply:
What, you mean like `PF_MANEUVER_NO_DAMP` and `PF_AI_NO_DAMP`?  That seems awkward, since physics should be independent of AI.

Is there no way that `sexp_player_use_ai` can conditionally remove that flag?

Reply:
Not easily, there's no way to know how the flag was applied, either due to an active maneuver or some other method. If you, say, wanted a ship to fundamentally never have damping via this flag, you could simply add a conditional that would preclude it from removing it if it had that ship flag. But having it as a separate flag would mean you wouldn't have to remember those places and accordingly update them.

Reply:
Mmh.  That would be hacky, but probably the most efficient solution.